### DISCUSSION TITLE
How to install Bladebit in Arch Linux?

### DISCUSSION THREAD

**Discussion by Sonix1** [Chia-Network/bladebit#264] [Q&A] [answered]
Can anyone help me with  Bladebit installation on Arch Linux?
I'm newbie in Linux. I tried but there is a problem with the libraries (different then Ubuntu) and compiling the project.
Thanks in advance.

**Comment by wallentx** [comment, marked as answer]
To install the prereq packages:
`sudo pacman -S base-devel cmake gmp numactl`
To build from source, navigate to some dir where you might clone repos (I use `~/src`), and 
`git clone https://github.com/Chia-Network/bladebit.git && cd bladebit && git checkout develop && sh build.sh`

Once it finishes compiling, you'll have a new directory called `build-release`, and the bladebit binary will be located in there.
You can either run it directly from that dir, or symlink it to some other dir within your `PATH`

**Reply by Sonix1** [reply]
Thank you!
